src/content/docs/more-integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry.mdx

```diff
@@ -1,31 +1,150 @@
 ---
-title: Micrometer metrics registry
-tags:
-  - Integrations
-  - Open source telemetry integrations
-  - Micrometer
+title: Forward Micrometer data to New Relic with OpenTelemetry
 metaDescription: New Relic offers an integration that sends your Micrometer telemetry data to your New Relic account.
 redirects:
-  - /docs/integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry
-  - /docs/integrations/open-source-telemetry-integrations/open-source-telemetry-integration-list/new-relics-micrometer-integration
+    - /docs/integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry
+    - /docs/integrations/open-source-telemetry-integrations/open-source-telemetry-integration-list/new-relics-micrometer-integration
 freshnessValidatedDate: never
 ---
 
-[Micrometer](https://micrometer.io/) provides a simple facade over the instrumentation clients for the most popular monitoring systems, allowing you to instrument your JVM-based application code without vendor lock-in.
+You can configure the OpenTelemetry bridge to forward Micrometer data to New Relic. The OpenTelemetry Micrometer bridge directs Micrometer to record telemetry to OpenTelemetry, which you can then export to New Relic with the OpenTelemtry SDK. 
 
-New Relic's Micrometer metrics registry sends your Micrometer telemetry data to your New Relic account. Don't have one? [Sign up for free](https://newrelic.com/signup). No credit card required.
+This doc walks you through procedures to forward Micrometer data with an example Spring app. These are generalized steps that you may want to update to fit your specific environment. 
 
-Resources for our Micrometer integration:
+## Compatibility and requirements [#requirements]
 
-* [README](https://github.com/newrelic/micrometer-registry-newrelic)
-* [Integration specs](https://github.com/newrelic/newrelic-exporter-specs/tree/master/micrometer)
+Before you get started, we recommend that review the requirements outlined in the [Getting started](https://github.com/open-telemetry/opentelemetry-java-instrumentation?tab=readme-ov-file#getting-started) section of the OpenTelemetry Java instrumentation repo. Once you've reviewed this section, return to this doc.
 
-<InstallFeedback />
+At minimum, you should:
 
-## Find your data
+* Have a Springboot app with Micrometer enabled. Your Springboot app should be instrumented with Micrometer out of the box.
+* Ensure your OpenTelemetry Java agent is the latest version.
```

Comment:
- Configure the OpenTelemetry Metrics SDK to export data to New Relic via OTLP
- Install the OpenTelemetry Micrometer bridge